man/man4/zfs.4

```diff
@@ -831,6 +831,13 @@ even with a small average compressed block size of ~8 KiB.
 The parameter can be set to 0 (zero) to disable the limit,
 and only applies on Linux.
 .
+.It Sy zfs_arc_shrinker_seeks Ns = Ns Sy 2 Pq int
+Relative cost of ARC eviction on Linux, AKA number of seeks needed to
+restore evicted page.
+Bigger values makes ARC more precious and evictions smaller, comparing to
+other kernel subsystems.
+Value of 4 means parity with page cache.
```

Comment:
Considering this affects amount of eviction by order of magnitude, I can hardly believe why it should be hard-coded.  The other question is that thanks to zfs_arc_shrinker_limit we really for the most part ignore what kernel wants from us.  Partially it is because kernel at times is completely insane in its requests, and new mentioned Multi-Gen LRU brings it to extreme, sometimes requesting eviction quarter to half of ARC for no reason.  I've already sent complaint email to the author, hope it lead to something.